package.json

```diff
@@ -57,54 +58,60 @@
     "autoprefixer": "10.4.15",
     "axe-core": "4.6.3",
     "babel-loader": "8.3.0",
+    "change-case": "^4.1.2",
     "cheerio": "1.0.0-rc.10",
     "chokidar": "3.5.3",
     "chromatic": "6.19.9",
     "concurrently": "7.6.0",
     "conventional-changelog-conventionalcommits": "5.0.0",
-    "cpy": "10.0.0",
     "cpy-cli": "5.0.0",
+    "cpy": "10.0.0",
     "dedent": "0.7.0",
-    "eslint": "8.44.0",
     "eslint-config-prettier": "8.8.0",
     "eslint-plugin-import": "2.27.5",
     "eslint-plugin-jest": "27.2.3",
     "eslint-plugin-jsdoc": "46.4.6",
     "eslint-plugin-prettier": "4.2.1",
     "eslint-plugin-react": "7.33.2",
     "eslint-plugin-unicorn": "46.0.1",
+    "eslint": "8.44.0",
+    "gh-release": "^7.0.2",
     "husky": "8.0.3",
-    "jest": "27.4.5",
     "jest-axe": "7.0.1",
     "jest-cli": "27.4.5",
+    "jest": "27.4.5",
```

Comment:
we've had `jest` in our "do not update list" since before I joined the team. @jcfranco @driskull @paulcpederson do any of you remember why it's there?

Reply:
I'm pretty sure we need to keep it locked to whatever stencil.js uses, thats why.

Reply:
Correct. Stencil dictates the versions of `jest` and other core packages (e.g., `puppeteer`).